test/compose/history/rc

```diff
@@ -0,0 +1,7 @@
+# Make our expansion have a predictable timestamp
+hook global ClientClose .* %{
+    evaluate-commands %sh{
+        kak -f 'ghf<space>ec$timestamp<esc>2f<space>ec$timestamp<esc>' <kak_quoted_history >tmp
+        mv tmp kak_quoted_history
```

Comment:
just pass kak_quoted_history as an argument to `kak -f` to edit the file in place, no need for the tmp dance.